### ISSUE TITLE
Contact Us Section.

### ISSUE THREAD

**Issue by utkarsh2102** [UdacityFrontEndScholarship/udabuddy#101]
How about the "Contact Us" section at the bottom of https://udbuddy.herokuapp.com/ ?
It'd have social media links of the projects and would also contain the "fork me" section at the corner.
If we have a mutual consent, I can work on the same and make a PR?

**Comment by littlestar642** [comment]
That's a cool idea. I guess we can have such footers for every page there in our UI. Please show me a mockup or if you prefer, a deployment of a design.

**Comment by utkarsh2102** [comment]
Sure @littlestar642.
I'll update you with something basic today. If you like the idea overall, we can then work on the colour blending as well.

**Comment by littlestar642** [comment]
@utkarsh2102 I guess having a footer with sectors like quick links ,contact us and license will be a better use of the display,rather than a single segment. If you feel the same, make a design and make a pr,as i can see you know about minimal design. Else do tell here your thoughts

**Comment by utkarsh2102** [comment]
@littlestar642,
Please check this footer :
![footer](https://user-images.githubusercontent.com/31288318/39825700-1d57c60c-53d0-11e8-855f-2777786ad8e3.png)
Let me know if you like it?

**Comment by littlestar642** [comment]
Yeah,looks good @utkarsh2102 . Decide the text in reference to the app. For ex- starting left to right
Udabuddy                                              Quick links                 Stay connected           This last columns
FAQ's(a page deemed to be made)       Links to pages           facebook and other     has to be thought of 
conatact us(gmail,i guess)                     of our present app      social media icons      because of now we  
                                                                                                                                      have not decided of  
                                                                                                                                      a mailing list

**Comment by littlestar642** [comment]
@utkarsh2102 Any ETA on this?

**Comment by sanchit04** [comment]
can i try this? @littlestar642

**Comment by littlestar642** [comment]
@sanchit04 I guess @utkarsh2102 is busy. You can surely go ahead making this footer section

**Comment by sanchit04** [comment]
what things should i add in the footer? can you give me some details.. @littlestar642

**Comment by littlestar642** [comment]
The image i provided here can be taken as a reference. You can add any other feature, but please inform once

**Comment by utkarsh2102** [comment]
The PR has been made. 
CC: @littlestar642 @sanchit04

**Comment by littlestar642** [comment]
The issue is still open. If someone wishes, can take this up!. Please inform here for the same.

**Comment by vsuguna** [comment]
Can I take this?

**Comment by littlestar642** [comment]
Sure you can, I guess this will be easy for you because you already know partials @souryavaranasi
